### PR TITLE
HP-654

### PR DESCRIPTION
**Pull request by rellfy**
Adds support to the arbitrum testnet

### CODE REVIEW

**Review by xiangzhe0812 — APPROVED** [review]

**Review by ahinchliff — APPROVED** [review]

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+﻿version: '2'
+services:
+  nginx-proxy:
+    image: nginxproxy/nginx-proxy
+    ports:
+      - "80:80"
```
Comment (bitcoinbrisbane) [inline docker/docker-compose.yml:6]: Do we need 80?

**Review by bitcoinbrisbane — COMMENTED** [review]
Should we lint the compose file? Seems inconsistent quotes

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      - /var/run/docker.sock:/tmp/docker.sock:ro
+  nginx-proxy-acme:
+    image: nginxproxy/acme-companion
+    volumes_from:
+      - nginx-proxy:rw
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+      - ./acme:/etc/acme.sh
+    environment:
+      DEFAULT_EMAIL: 'support@handle.fi'
```
Comment (bitcoinbrisbane) [inline docker/docker-compose.yml:21]: Double quotes?

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      - nginx-proxy
+  graph-node:
+    image: graphprotocol/graph-node
+    ports:
+      - "8001:8001"
+      - "8020:8020"
+      - "8030:8030"
+      - "8040:8040"
+    expose:
+      - '8000'
```
Comment (bitcoinbrisbane) [inline docker/docker-compose.yml:32]: Double quotes?

File: docker/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  ipfs:
+    image: ipfs/go-ipfs:v0.4.23
+    ports:
+      - '5001:5001'
+    volumes:
+      - ./data/ipfs:/data/ipfs
+  postgres:
+    image: postgres
+    ports:
+      - '5432:5432'
```
Comment (bitcoinbrisbane) [inline docker/docker-compose.yml:58]: And here

File: docker/docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+      ethereum: 'rinkebyarbitrum:https://arb-rinkeby.g.alchemy.com/v2/Xp6EK9frm2n1D0Nz6PY-69AjX5uFReKe'
+      GRAPH_LOG: info
+      VIRTUAL_PORT: 8000
+      VIRTUAL_HOST: 'rinkebyarbitrum.subgraph.handle.fi'
+      LETSENCRYPT_HOST: 'rinkebyarbitrum.subgraph.handle.fi'
+      LETSENCRYPT_EMAIL: 'support@handle.fi'
+  ipfs:
+    image: ipfs/go-ipfs:v0.4.23
+    ports:
+      - '5001:5001'
```
Comment (bitcoinbrisbane) [inline docker/docker-compose.yml:52]: And here

File: docker/docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+      - ipfs
+      - postgres
+      - nginx-proxy-acme
+    environment:
+      postgres_host: postgres
+      postgres_user: graph-node
+      postgres_pass: let-me-in
+      postgres_db: graph-node
+      ipfs: 'ipfs:5001'
+      ethereum: 'rinkebyarbitrum:https://arb-rinkeby.g.alchemy.com/v2/Xp6EK9frm2n1D0Nz6PY-69AjX5uFReKe'
```
Comment (bitcoinbrisbane) [inline docker/docker-compose.yml:43]: And theses

File: docker/docker-compose.yml (hunk 1 above)
Comment (rellfy) [reply docker/docker-compose.yml:6]: I believe it is needed for the letsencrypt certificate challenge

**Pull request merged**